.github/workflows/review.yaml

```diff
@@ -0,0 +1,52 @@
+
+name: PR Daily
+
+on:
+  schedule:
+    - cron: "10 1 * * *" # once a day
+
+  pull_request:
+    # test PRs targeting this branch code
+    branches: [ "master" ]
+
+  # allows to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  # Cancel ongoing tests in case of push to the same PR or staging branch,
+  # but let previous master commit tests complete.
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+env:
+  # empty except for pull_request events
+  PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+
+  # Full clones of Squid repository branches (depth=19000+) waste resources,
+  # while excessively shallow clones break tests that check for past commits
+  # (e.g., to skip a particular test until a known bug is fixed) or generate
+  # diffs against them (e.g., for `git diff --check`). This arbitrary limit
+  # tries to balance the two concerns.
+  CHECKOUT_FETCH_DEPTH: 1001
+
+jobs:
+
+  hold-for-review:
+    # Alex Rousskov has been granted a 48hr delay
+    # to allow him a chance to veto new additions
+    # regardless of other Squid Project member votes.
```

Comment:
Wouldn't everyone in the approver list get the same privilege? I don't see anything special in the code about Alex

Reply:
Well, he requested it and is the only one of us making use of it vs the older process of reverting changes.

Reply:
Reworded in 78ba5c7caf anyway.